client/src/components/pages/RobotFile/RobotFile.jsx

```diff
@@ -2,7 +2,6 @@ import React from 'react';
 import { Layout } from 'antd';
 import HeaderNavbar from '../../content/HeaderNavbar/HeaderNavbar';
 import { Typography } from 'antd';
```

Comment:
Can Typography and Layout be imported in the same statement?

Reply:
yes, good point. Done.